app/javascript/mastodon/features/notifications/components/clear_column_button.js

```diff
@@ -17,9 +13,15 @@ class ClearColumnButton extends React.Component {
     const { intl } = this.props;
 
     return (
-      <div role='button' title={intl.formatMessage(messages.clear)} className='column-icon column-icon-clear' tabIndex='0' onClick={this.props.onClick}>
+      <button
+        className='text-btn column-header__setting-btn'
+        tabIndex='0'
+        onClick={this.props.onClick}
+      >
         <i className='fa fa-eraser' />
-      </div>
+        {' '}
```

Comment:
`<i />` and `<FormattedMessage />` can be on the same line with a space between them